src/core/meshcop/border_agent.cpp

```diff
@@ -873,9 +866,10 @@ void BorderAgent::ClearEphemeralKey(void)
     case kStateStopped:
     case kStateConnected:
```

Comment:
here may explicitly add comment about TIMEOUT_COMM_PET for `connected` case?

Reply:
Updated in new push.